options/options.go

```diff
@@ -79,10 +80,18 @@ type TerragruntOptions struct {
 }
 
 // Create a new TerragruntOptions object with reasonable defaults for real usage
-func NewTerragruntOptions(terragruntConfigPath string) *TerragruntOptions {
+func NewTerragruntOptions(terragruntConfigPath string) (*TerragruntOptions, error) {
 	workingDir := filepath.Dir(terragruntConfigPath)
 
-	downloadDir := filepath.Join(os.TempDir(), "terragrunt")
+	logger := util.CreateLogger("")
+
+	homedir, err := homedir.Dir()
+	if err != nil {
+		logger.Fatalf("error: %v\n", err)
```

Comment:
So... From the [FatalF docs](https://golang.org/pkg/log/#Fatalf):

```
Fatalf is equivalent to Printf() followed by a call to os.Exit(1).
```

Did I ever tell you that Go is a shitty language?

Reply:
Use a `PrintF`. In a separate PR, we should move Terragrunt off the default Go logging library and onto something that has proper `Info`, `Error`, etc methods.